integration-tests/src/test/java/oracle/weblogic/kubernetes/ItTwoDomainsLoadBalancers.java

```diff
@@ -500,7 +583,7 @@ public void testApacheDefaultSample() {
    * For more details, please check:
    * https://github.com/oracle/weblogic-kubernetes-operator/tree/master/kubernetes/samples/charts/apache-samples/custom-sample
    */
-  @Order(9)
+  @Order(12)
   @Test
   @DisplayName("verify Apache load balancer custom sample through HTTP and HTTPS channel")
   public void testApacheCustomSample() {
```

Comment:
Modify Test name to testApacheLoadBalancingCutomSample()

Reply:
fixed